packages/trace-viewer/vite.sw.config.ts

```diff
@@ -36,7 +36,7 @@ export default defineConfig({
     },
   },
   build: {
-    outDir: path.resolve(__dirname, '../playwright-core/lib/vite/traceViewer'),
+    outDir: path.resolve(__dirname, 'public'),
```

Comment:
we need this so that the dev server can serve the service worker.

Reply:
who writes now to this folder?

Reply:
The main Vite build that builds the trace viewer: https://github.com/microsoft/playwright/pull/33228/files#diff-7f0209d537fec3cec4cb343bba8c4d2734b101e57c1f339f1c8009e8c5aa123cR43

Reply:
How is this going to work in production if trace viewer sw is not bundled inside playwright-core?